index.html

```diff
@@ -91,28 +91,28 @@ <h2>A Note on Feature Detection</h2>
   </p>
   <aside class="example" title="Checking existence of ComputePressureObserver interface">
     <p>
-      This simple example illustrates how to check whether the User Agent exposes the ComputePressureObserver interface
+      This simple example illustrates how to check whether the User Agent exposes the
+      ComputePressureObserver interface
     </p>
     <pre class="js">
```

Comment:
I'd suggest having one way only to check. I like 

```js
if ("ComputePressureObserver" in globalThis) {
  // Create ComputePressureObserver interface
}
```

Reply:
I agree

Reply:
Then you need to update "All of these tell you something" below as well

Reply:
Great point, fixed